study.md

```diff
@@ -50,7 +50,7 @@ Does `proxima_centauri.print_mass` run? Why or why not? If so, then what is the
 output? If not, then how can it be fixed?
 
 ```md
-<!-- your answer here -->
+<!-- Again they will not run because the attributes are still set as private.  -->
```

Comment:
It won't run because `mass_kg` is a local variable that hasn't been defined.  One possible fix is to make `mass_kg` into an instance variable.